entity-framework/core/miscellaneous/cli/dbcontext-creation.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Design-time DbContext Creation - EF Core
+author: bricelam
+ms.author: bricelam
+ms.date: 10/27/2017
+ms.technology: entity-framework-core
+---
+Design-time DbContext Creation
```

Comment:
Do we need to explain what "design-time" is? (Maybe just (for example, when running Migrations commands). /cc @divega

Reply:
Added your example.